src/main/java/ca/bc/gov/api/oracle/legacy/util/ClientMapper.java

```diff
@@ -0,0 +1,21 @@
+package ca.bc.gov.api.oracle.legacy.util;
+
+import ca.bc.gov.api.oracle.legacy.dto.ClientPublicViewDto;
+import ca.bc.gov.api.oracle.legacy.entity.ClientPublicViewEntity;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class ClientMapper {
```

Comment:
Maybe we can plan on including a mapper dependency such as orika to deal with Dto <-> Entity conversions

Reply:
Will be done in future PR, when moving to webflux